src/stream/src/executor/managed_state/aggregation/mod.rs

```diff
@@ -31,7 +31,7 @@ use crate::executor::PkDataTypes;
 
 mod extreme;
 
-mod string_agg;
+// mod string_agg;
```

Comment:
remove this?

Reply:
Leave old `string_agg.rs` as might be referenced by @richardchien , will remove `string_agg.rs` later.

Reply:
I will remove this `mod` and file in later PR